src/doc/rustc/src/target-tier-policy.md

```diff
@@ -104,6 +104,29 @@ indicates something entirely optional, and does not indicate guidance or
 recommendations. This language is based on [IETF RFC
 2119](https://tools.ietf.org/html/rfc2119).
 
+## Adding a new target
+
+New targets typically start as Tier 3 and then can be promoted later.
+To propose addition of a new target, open a pull request on [`rust-lang/rust`]:
```

Comment:
I assumed that this link target exists, if it doesn't, can you add it?

Reply:
Added and checked that it is working. I also added an entry to the TOC with [87b2110](https://github.com/rust-lang/rust/commit/87b2110f25bed1407a9aded9e263b64dc3315d45).

Reply:
Thanks for catching it :)